Framework/PythonInterface/mantid/api/src/Exports/DetectorInfo.cpp

```diff
@@ -0,0 +1,14 @@
+#include "MantidAPI/DetectorInfo.h"
+#include <boost/python/class.hpp>
+
+using Mantid::API::DetectorInfo;
+using namespace boost::python;
+
+void export_DetectorInfo() {
+  class_<DetectorInfo, boost::noncopyable>("DetectorInfo", no_init)
+      .def("size", &DetectorInfo::size, (arg("self")),
```

Comment:
I think that `size` is too ambiguous going forward. The prototype has the ability to tell you via `DetectorInfo`: 

* The total number of components  
* The number of which are path components
* The number of which are actually detectors

I'm prefer we were a bit more explicit even though the natural assumption is that we are talking about detector size.